build.gradle

```diff
@@ -374,6 +374,7 @@ dependencies {
   implementation(group: 'org.springframework.security', name: 'spring-security-crypto', version: versions.springSecurityCrypto) {
     force = true
   }
+  implementation group: 'io.netty', name: 'netty-transport-native-epoll', version: '4.1.84.Final'
```

Comment:
is this required for this PR?

Reply:
This is required for Mac with M1 chip  otherwise local build will fail